guides/extensions/embedded-scripts/embedded-script-extension-files-and-code.md

```diff
@@ -0,0 +1,149 @@
+---
+articleType: reference
+tags: 
+  extensions: 
+    - embeddedScript
+  interfaces:
+    - site
+  frameworks: 
+    - cli
+---
+
+# Embedded Script Extension Files and Code
+
+The structure of an embedded script extension in a CLI app is as follows:
+
+  ```tsx
+  .
+  └── <your-app-name>/
+      └── src/
+          └── site/
+              └── embedded-scripts/
+                  └── <your-script-name>/
+                      ├── embedded.html
+                      ├── embedded.json
+                      └── params.dev.json
+  ```
+
+These files can be generated in your app by running the following command and selecting **Embedded Script extension**:
+
+```tsx
+npm run generate
+```
+
+## embedded.html
+
+The file named `embedded.html` contains the code you wish to inject. Your code will be added to the page's head or body depending on your configuration.
+
+Your file structure should look like this:
+
+In your HTML code, you can:
+- [Reference local files](#referencing-local-files-in-your-html-code)
+- [Use dynamic parameters](#using-dynamic-parameters-in-your-html-code)
+- [Add global CSS](#add-global-css-to-your-html-code)
+
+### Referencing local files in your HTML code
+
+Wix will host and deploy every file in your project unless you specify otherwise, including any that you add. Your HTML code can reference these files using a relative path. Your HTML code can reference these files using a relative path.
+
+When referencing local files in a `<script>` tag, the tag needs to have `type="module"`.
+
+For example, to reference a file named `local-script.js` in the same directory as `embedded.html`, use the following code:
+
+  ```tsx
+  <script type = "module" src = "./local-script.js"></script>
+  ```
+>**Note:** TypeScript files are supported.
+
+### Using dynamic parameters in your HTML code
+
+You can embed dynamic parameters in your code to inject custom information per site.
```

Comment:
I think it will be nice to have a more detailed explanation of "what are dynamic parameters".